tests/test_ide.py

```diff
@@ -0,0 +1,28 @@
+import json
+import yaml
+from . import TestGramex
+
+
+class TestIde(TestGramex):
+    def test_get(self):
+        url = '/ide_config_handler?filename=test_ide.yaml'
+        result = self.get(url).content
+        with open('test_ide.json', 'r') as fin:
+            text = json.load(fin)
+        self.maxDiff = None
+        self.assertEquals(text, json.loads(result.decode("utf-8")))
```

Comment:
Less is better than more. Can we:

- Use `text = gramex.cache.open('test_ide.json')`
- Remove `self.maxDiff = None`

Reply:
Done!